lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -190,6 +196,9 @@ async function runSmokeTest(testOptions) {
       localConsole.log(`Retrying run (${i} out of ${retries} retries)...`);
     }
 
+    // Ensure server only records URLs fetched for this run.
+    server.clearRequestUrls();
```

Comment:
could have `server.getRequestUrls()` also clear existing URLs and so be able to not have to manually call this

Reply:
`takeRequestUrls` if we do that